hts.c

```diff
@@ -74,7 +79,7 @@ const char *hts_version()
 }
 
 unsigned int hts_features(void) {
-    unsigned int feat = 0;
+    unsigned int feat = HTS_FEATURE_HTSCODECS; // Always present
```

Comment:
Having amended this, it should also get an amended test/test_introspection.c file to utilse the new versioning.

Reply:
Done, test_introspection now exercises this.